R/annotateFcsFile.R

```diff
@@ -16,14 +16,14 @@
 #' @export
 #' @examples
 #' \dontrun{
-#' annotations = list(list(name="annotations 1", value=1), list(name="annotation 2", value="myValue"))
+#' annotations <- list(list(name = "annotations 1", value = 1), list(name = "annotation 2", value = "myValue"))
```

Comment:
R CMD check complains about line length here now. Can you break it please? Don't forget to rebuild Rd files after.